modules/@apostrophecms/ui/ui/apos/lib/i18next.js

```diff
@@ -33,6 +33,10 @@ export default {
     // multiple layers of code, as a single `label`
     // property for instance
     Vue.prototype.$t = (phrase, options) => {
+      if (phrase == null) {
```

Comment:
Unrelated fix: don't log console turds if $t is passed null, as can happen if a button has no label, for instance.